src/researchhub_case/serializers/external_author_claim_case_serializer.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework.serializers import ModelSerializer
+
+from researchhub_case.models import ExternalAuthorClaimCase
+
+from .abstract_researchhub_case_serializer import DynamicAbstractResearchhubCase
+from .researchhub_case_abstract_serializer import EXPOSABLE_FIELDS
+
+
+class ExternalAuthorClaimCaseSerializer(ModelSerializer):
```

Comment:
Just a naming suggestion, but up to you whether you want to make it.

The way I understand it, the "case" is not "external", but rather the api call to fetch data is.
The case itself is an internal ResearchHub case. For consistency, I would just call it `AuthorClaimCaseSerializer`

Reply:
Yeah, I'd prefer a different name too, but `AuthorClaimCaseSerializer` is already used for claiming author profiles. So, I appended external as a prefix since `external` makes sense for author profiles retrieved from Scholarly (and potentially future external apis)